scripts/devops_tasks/Smoke-Tests-Nightly.ps1

```diff
@@ -0,0 +1,58 @@
+#Requires -Version 7.0
+
+Param(
```

Comment:
Can we add a section summarizing the script, what it does, and what it outputs?

Just above the `Param()` you need something like.

```powershell
.SYNOPSIS
<one liner>

.DESCRIPTION
<deeper description>

.PARAMETER ServiceFolder
The folder in which context the test proxy will be started.
```